aten/src/ATen/native/native_functions.yaml

```diff
@@ -2636,6 +2636,10 @@
   use_c10_dispatcher: full
   supports_named_tensor: True
   variants: function, method
+  dispatch:
+    CPU: tanh
+    CUDA: tanh
```

Comment:
do we need CUDA here? check other nearby entries CPU/CUDA all point to different implementation...

Reply:
I am not 100% familiar with the `native_functions.yaml`. Originally it didn't have explicit `dispatch`, which (I assumed) would point to default `tanh` implementation for all devices. However, because I am adding `quantized_tanh`, I have to specify the `dispatch` explicitly. That's why I put the both the CPU and CUDA in there

Reply:
please test to ensure that cuda functionality for tanh is also tested in this case after the changes